openintel/settings.py

```diff
@@ -116,6 +116,10 @@ def _detect_command(cmd):
     "notifications",
     # invitations must appear after allauth: https://github.com/bee-keeper/django-invitations#allauth-integration
     "invitations",
+    # django-filters for filtering
+    "django_filters",
+    "bootstrap4",
```

Comment:
Where does bootstrap4 actually end up getting used? I didn't see a `{% load bootstrap4 %}` anywhere in the templates

Also, the project is currently on bootstrap3. I think bootstrap 3 uses `ul` and `li` for dropdowns instead of `div`s:
https://getbootstrap.com/docs/3.3/components/#dropdowns

With bootstrap 5 coming out relatively soon, will actually probably jump over issue https://github.com/twschiller/open-synthesis/issues/244 to add bootstrap 4 styling directly to bootstrap 5 at somepoint